graphql/resolvers/User.js

```diff
@@ -67,12 +75,38 @@ module.exports = {
     }
     return null
   },
-  portrait (user, args, { user: me }) {
+  portrait (user, args, { user: me, req }) {
     if (canAccessBasics(user, me)) {
-      const { portraitUrl } = user._raw
-      return portraitUrl
-        ? getImageUrl(portraitUrl, args)
-        : portraitUrl
+      let { portraitUrl } = user._raw
+      if (!portraitUrl) {
+        return portraitUrl
+      }
+      // TODO migrate DB entries from _384x384 to _original and from ASSETS_BASE_URL to ASSETS_SERVER_BASE_URL/s3/AWS_S3_BUCKET
+      const legacyMode = portraitUrl.indexOf('_384x384') > -1
+      if (legacyMode) {
+        portraitUrl = portraitUrl.replace('_384x384', '_original')
+        portraitUrl = portraitUrl.replace(ASSETS_BASE_URL, `${ASSETS_SERVER_BASE_URL}/s3/${AWS_S3_BUCKET_PROD || AWS_S3_BUCKET}`)
```

Comment:
For a truly smooth transition we'd need to use the proxy endpoint here. Because we will never be able to sync s3 transition and backend deploy—can't turn off the old bucket until backend is deployed. Could get new content while the backend is deploying.